DEV-GUIDE.md

```diff
@@ -209,6 +224,15 @@ Push the containers to your registry:
 make push
 ```
 
+### Building containers with debugger information
+
+To build all the Istio containers with debugger information, issue the following
+command:
+
+```shell
+make docker DEBUG=1
```

Comment:
I'm used to seeing the variables placed first, e.g., "make DEBUG=1 docker", even though either way seems to work.

Presently I think the "make docker" won't actually work on a clean tree since the dependencies aren't explicit and mixer's push-docker and Makefile just go straight to invoking "docker build" without first running "go build". IOW, it only works if other make targets were run previously to create some of the go executables.  Consequently, "make DEBUG=1 docker" might not really, as claimed here, build all containers with debugger information (the results of the mixer-related containers depends on how mixer was previously compiled).

I'm not sure if I'm suggesting that the description be clarified, the build system be improved, or the example be dropped for now.

Reply:
I'll change to "make DEBUG=1 docker"

Looks like it's working since push-docker is also calling bin/gobuild.sh. From I can see when issuing the above, go build is called with the correct flags. Let me know otherwise.

Reply:
I think I was overlooking that mixer's Makefile has the docker target depend on docker/mixs, and this will cause a build.